Meshery-Adapters/kuma-meshery-adapter/index.json

```diff
@@ -38,11 +38,11 @@
     "hideHiddenFiles": true,
     "exclusionPatterns": ["*test", "./test/*", "./logs"],
     "showdashboard": true,
-    "dashboards": [{"name": "Meshery Server", "port": 9081}],
+    "dashboards": [{"name": "Meshery UI", "port": 9081},{"name": "Image Hub", "port": 8080}],
     "uilayout": "terminal-iframe"
   }, 
 
   "backend": {
-    "imageid": "kubernetes-cluster-running:1.18"
+    "imageid": "kubernetes-cluster-running:1.22"
```

Comment:
we took this image from the katacoda library, are you sure this works?

Reply:
It was suggested by @piyushsingariya. And it is the only line of code I saw that mentioned the version of Kubernetes. So I guess it should work 🤞

Reply:
Yes, the docs only mention about 1.18 version. What should we do? The other adapters will work fine but OSM requires kubernetes >=1.19.

Reply:
Should we disable OSM lab for some time until we figure some solution out and keep the other labs running with the Kubernetes version 1.18?

Reply:
@leecalcote @warunicorn19 thoughts on this approach?

Reply:
yes, I think we should remove the OSM lab for the time being, since it doesn't work and I don't see a workaround either. @leecalcote ?

Reply:
The trafeik-mesh adapter too. It's still not connecting after the install command.

Reply:
I don't know that the OSM installations are working in the Meshery Adapter for OSM even on later K8s versions. @Aisuko, @Revolyssup, is it?